signals.mss

```diff
@@ -240,6 +240,33 @@ Format details:
     marker-allow-overlap: true;
   }
 
+  /*********************************************/
+  /* DE wrong road signal Zs 6(DB) /Zs 7(DR)  */
+  /*********************************************/
+  /* sign */
+  [zoom>=17]["wrong_road"="DE-ESO:zs6"]["wrong_road_form"="sign"] {
+    marker-file: url('symbols/de/zs6-sign.svg');
+    marker-width: 7;
+    marker-height: 16;
```

Comment:
Ratio of width and height does not match the ratio of the SVG file:

7/16 = 0.4375
159/205 = 0.7756

Result:

![zs6-18-bad-hersfeld-ausschnitt](https://user-images.githubusercontent.com/3611273/197338549-0b692da1-cddf-415b-b8f1-e2af9c148bb9.png)